lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -194,7 +207,7 @@ class FeatureCard extends React.Component {
               <h2 className="heading -delta">{campaign.title}</h2>
               <p className="tagline">{campaign.tagline}</p>
               <ul className="form-actions">
-                {(campaign.signedUp && !this.state.isSignedUp) ? <h1>You're signed up already!</h1> : <li><button className={`${this.state.isSignedUp ? 'tada' : ''} button`} onClick={()=>this.signup()}>Sign Up</button></li>}
+                <li><Button classes={['-reactive']} text="Sign Up" loading={this.state.isLoading} disabled={this.state.actionDisabled} success={this.state.isSignedUp} onClick={this.signup}></Button></li>
                 <li><a className="button -tertiary" onClick={()=>this.viewAnother(true)}>Show me another</a></li>
```

Comment:
it can't cause it's supposed to be an anchor link and not a button... unless, it doesn't need to be...